src/content/en/2020/http2.md

```diff
@@ -403,69 +632,174 @@ Looking at the frequency of push by content type in Figure 22.20, we see 89% of
+        <td class="numeric">3</td>
+        <td class="numeric">2</td>
+        <td class="numeric">2</td>
+        <td class="numeric">1</td>
+      </tr>
+      <tr>
+        <td class="numeric">50</td>
+        <td class="numeric">4</td>
+        <td class="numeric">14</td>
+        <td class="numeric">4</td>
+        <td class="numeric">4</td>
+        <td class="numeric">4</td>
+        <td class="numeric">1</td>
+      </tr>
+      <tr>
+        <td class="numeric">75</td>
+        <td class="numeric">16</td>
+        <td class="numeric">20</td>
+        <td class="numeric">4</td>
+        <td class="numeric">11</td>
+        <td class="numeric">8</td>
+        <td class="numeric">1</td>
+      </tr>
+      <tr>
+        <td class="numeric">90</td>
+        <td class="numeric">29</td>
+        <td class="numeric">32</td>
+        <td class="numeric">4</td>
+        <td class="numeric">24</td>
+        <td class="numeric">23</td>
+        <td class="numeric">1</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Distribtion of number pushed asset by content type on mobile.", sheets_gid="238923402", sql_file="number_of_h2_and_h3_pushed_resources_and_bytes_by_content_type.sql") }}</figcaption>
 </figure>
 
-Pushing of video dominates the higher percentiles of bandwidth use. This can however really negatively impact page performance by contending with the other resources required to render the page.
+Pushing of video dominates the higher percentiles of bandwidth use. This can, however, negatively impact page performance by contending with the other resources required to render the page.
 
-<figure markdown>
-|  Percentile   | css   | font   | html | image  | script | video  |
-| ------------- | ----- | ------ | ---- | ------ | ------ | ------ |
-| 10            | 21.1  | 784.7  | 1.5  | 4.1    | 4.0    | 3444.3 |
-| 25            | 51.0  | 811.6  | 23.0 | 9.7    | 36.4   | 3444.3 |
-| 50            | 105.7 | 916.0  | 78.2 | 61.3   | 165.5  | 3444.3 |
-| 75            | 204.2 | 983.6  | 81.1 | 501.2  | 344.8  | 3444.3 |
-| 100           | 316.3 | 1237.6 | 83.3 | 2770.7 | 603.0  | 3444.3 |
-
-<figcaption>{{ figure_link(caption="Distribtion of pushed asset sizes (KB) by content type on mobile.", sheets_gid="238923402", sql_file="number_of_h2_and_h3_pushed_resources_and_bytes_by_content_type.sql") }}</figcaption>
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Percentile</th>
+        <th>css</th>
+        <th>font</th>
+        <th>html</th>
+        <th>image</th>
+        <th>script</th>
+        <th>video</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td class="numeric">10</td>
+        <td class="numeric">21.1</td>
+        <td class="numeric">784.7</td>
+        <td class="numeric">1.5</td>
+        <td class="numeric">4.1</td>
+        <td class="numeric">4.0</td>
+        {# TODO(analysts): Please double check the video stats, which is 3444.3 for all percentiles. #}
```

Comment:
@bazzadp

Reply:
Looks to me like only one site pushes video. This means where video is pushed (that one site), all percentiles push the same amount.

Not sure how best to account for that. Any ideas @dotjs or @rviscomi ?

Reply:
I had the same initial confusion. As mentioned if only a single website is pushing video then the value is the same across all percentiles. I could add a sentence to explain ?

Reply:
Removed section. On reading again I don't think it adds much.